citations-api/api/src/java/org/sakaiproject/citation/api/CitationService.java

```diff
@@ -229,15 +229,15 @@
 	 */
 	public CitationCollectionOrder getNestedCollection(String citationCollectionId);
 	/**
-	 ** Removes the section of a citation collection and its children
+	 ** Removes the section of a citation collection and its children or removes a citation
 	 * @param collectionId
 	 * @param locationId
 	 */
-	public void removeSection(String collectionId, int locationId);
+	public void removeSectionOrCitation(String collectionId, int locationId);
```

Comment:
`removeLocation()` as a name as it doesn't really care what's at the location does it?